test/README

```diff
@@ -18,7 +18,7 @@ Testing requires phantomjs inside the cockpit package.
 To run the integration tests run the following. Don't run the integration tests
 as root:
 
-    $ ./VERIFY
+    $ ./test/verify/run-tests
 
 The first time you run the tests, they will create a cockpit1 bridged network used
```

Comment:
@dperpeet Is this paragraph correct?  I dont think `./test/verify/run-tests` downloads images. I'm not sure.